Makefile

```diff
@@ -63,10 +63,11 @@ test-e2e: ## Run openshift specific e2e test
 	# failures and flakes.
 	# Feature:Operator tests remove deployments. Thus loosing all the logs
 	# previously acquired.
-	hack/ci-integration.sh $(GINKGO_ARGS) -p -focus="Feature:Operators" || (hack/junitmerge.sh && exit 1)
-	hack/ci-integration.sh $(GINKGO_ARGS) -p -skip="Feature:Operators|Autoscaler" || (hack/junitmerge.sh && exit 1)
+	hack/ci-integration.sh $(GINKGO_ARGS) -p -focus="Feature:Operators" -skip="Serial" || (hack/junitmerge.sh && exit 1)
+	hack/ci-integration.sh $(GINKGO_ARGS) -p -skip="Feature:Operators|Autoscaler|Serial" || (hack/junitmerge.sh && exit 1)
 	# TODO: parallelise autoscaler
-	hack/ci-integration.sh $(GINKGO_ARGS) -focus="Autoscaler" || (hack/junitmerge.sh && exit 1)
+	# Run serial tests only
+	hack/ci-integration.sh $(GINKGO_ARGS) -focus="Autoscaler|Serial" || (hack/junitmerge.sh && exit 1)
```

Comment:
i think we should leave the autoscaler test line the way it was, those tests need more cleanup before we can start running them in parallel.

Reply:
This could gradually increase overall CI running time, I left the `TODO` as a reminder to do it later.

Reply:
Oh, they are both serial... Ok, I'll change.